File: lang_java/Ram.java
class Ram extends Komponen{

    private int kapasitasGB;
    private String ddr;

    public Ram(){
    }

    public Ram(String merk, String nama, int kapasitasGB, String ddr){
        super(merk, nama);
        this.kapasitasGB = kapasitasGB;
        this.ddr = ddr;
    }

    public void setKapasitasGB(int kapasitasGB){
        this.kapasitasGB = kapasitasGB;
    }

    public int getKapasitasGB(){
        return this.kapasitasGB;
    }

    public void setDdr(String ddr){
        this.ddr = ddr;
    }

    public String getDdr(){
        return this.ddr;
    }

}

File: lang_py/Ram.py
from Komponen import Komponen

class Ram(Komponen):
    def __init__(self, merk, nama, kapasitasGB, ddr):
        super().__init__(merk, nama)
        self.kapasitasGB = kapasitasGB
        self.ddr = ddr

    def setKapasitasGB(self, kapasitasGB):
        self.kapasitasGB = kapasitasGB

    def getKapasitasGB(self):
        return self.kapasitasGB
    
    def setDdr(self, ddr):
        self.ddr = ddr

    def getDdr(self):
        return self.ddr